editor/editor_properties_array_dict.cpp

```diff
@@ -517,6 +543,33 @@ void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_d
 		emit_changed(get_edited_property(), array, "", false);
 		update_property();
 	}
+
+	if (drag_data.has("type") && String(drag_data["type"]) == "nodes") {
+		Array nodes = drag_data["nodes"];
+		Array nodes_paths = Array();
```

Comment:
```suggestion
		Array node_paths;
```
Unnecessary, also grammar